iModelCore/ECDb/Tests/NonPublished/ConcurrentQueryTest_V2.cpp

```diff
@@ -1054,7 +1054,33 @@ TEST_F(ConcurrentQueryFixture, BlobIO) {
         ASSERT_STREQ(resp->GetError().c_str(), "BlobIO: offset + length provided is greater then size of blob");
     }
 }
+//---------------------------------------------------------------------------------------
+//@bsimethod
+//+---------------+---------------+---------------+---------------+---------------+------
+TEST_F(ConcurrentQueryFixture, CommentAtEndOfECSql) {
+    ASSERT_EQ(BentleyStatus::SUCCESS, SetupECDb("CommentAtEndOfECSql.ecdb", SchemaItem(
+        R"xml(<ECSchema schemaName="TestSchema" alias="ts" version="1.0" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.1">
+                <ECEntityClass typeName="testEntity">
+                    <ECProperty propertyName="entity_id" typeName="int" />
+                </ECEntityClass>
+            </ECSchema>)xml")));
+
+    ECSqlStatement stmt;
+    ASSERT_EQ(ECSqlStatus::Success, stmt.Prepare(m_ecdb, "INSERT INTO ts.testEntity(entity_id) VALUES(?)"));
+    stmt.BindInt(1, 1);
+    ASSERT_EQ(stmt.Step(), BE_SQLITE_DONE);
+    m_ecdb.SaveChanges();
+
+    auto& mgr = ConcurrentQueryMgr::GetInstance(m_ecdb);

```

Comment:
write a second test case in this test to cover both printf statements in the source code

Reply:
got it